assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/formDecorators/utilities.test.ts

```diff
@@ -0,0 +1,139 @@
+import { getPriceType, getPriceModifierPriceTypeGuid } from './utilities';
+import { nodes as priceTypes } from '../../../../services/apollo/queries/priceTypes/test/data';
+import { nodes as prices } from '../../../../services/apollo/queries/prices/test/data';
+import { TpcPriceModifier } from '../types';
+
+describe('getPriceType', () => {
+	it('should return the correct priceType object', () => {
+		const getPriceTypeForPrice = getPriceType(priceTypes);
+		// assuming that in test data
+		// number of priceType nodes is >= number of price nodes
+		prices.forEach((price, index) => {
+			// get test priceType
+			const testPriceType = priceTypes[index];
+			// update priceModifier using test priceType
```

Comment:
Great job on adding these comments. I need to do this in some more complex area of the code I do write.
:+1: